src/query_error.c

```diff
@@ -95,10 +112,18 @@ void QueryError_MaybeSetCode(QueryError *status, QueryErrorCode code) {
   status->code = code;
 }
 
-const char *QueryError_GetError(const QueryError *status) {
+const char *QueryError_GetUserError(const QueryError *status) {
```

Comment:
Can't we just use `QueryError_GetDisplayableError(status, false)`?

Reply:
Dvir asked for there to be two separate functions.